src/django_components/component.py

```diff
@@ -535,8 +541,10 @@ def _render_impl(
 
             with context.update(
                 {
+                    # Private context fields
                     _ROOT_CTX_CONTEXT_KEY: self.outer_context,
                     _FILLED_SLOTS_CONTENT_CONTEXT_KEY: updated_slots,
+                    _REGISTRY_CONTEXT_KEY: self.registry,
```

Comment:
Most of the changes were about wiring the registry instance to the component and slot, so we use the settings as defined on the registry, eg. `registry.settings.XXX`, instead of `app_settings.XXX`